src/views/stories/Container.tsx

```diff
@@ -346,6 +350,46 @@ export default function Container({
                                                   )}
                                         </Text>
                                     </Pressable>
+                                    {media && (
+                                        <Pressable
+                                            hitSlop={15}
+                                            style={{
+                                                flexDirection: 'row',
+                                                width: '100%',
+                                                alignItems: 'center',
+                                                justifyContent: 'flex-start',
+                                                marginLeft: 24,
+                                                marginVertical: 12,
+                                            }}
+                                            onPress={() => {
+                                                FileSystem.downloadAsync(
+                                                    media.url,
+                                                    FileSystem.documentDirectory +
+                                                        '.jpeg'
+                                                )
+                                                    .then(({ uri }) => {
+                                                        Sharing.shareAsync(uri);
+                                                        Clipboard.setString(
+                                                            `https://www.impactmarket.com/communities/${community.id}`
+                                                        );
+                                                    })
+                                                    .catch(() => {
+                                                        // TODO: some error occurred
```

Comment:
I'd like to avoid this. Even if was just an Alert approach, is better than nothing ...